novawallet/Modules/Vote/Governance/TinderGov/TinderGovViewModelFactory.swift

```diff
@@ -3,20 +3,24 @@ import Foundation
 protocol TinderGovViewModelFactoryProtocol {
     func createTinderGovReferendumsSection(
         for referendums: [ReferendumLocal]?,
+        accountVotes: ReferendumAccountVotingDistribution?,
         locale: Locale
     ) -> ReferendumsSection?
 }
 
 struct TinderGovViewModelFactory: TinderGovViewModelFactoryProtocol {
     func createTinderGovReferendumsSection(
         for referendums: [ReferendumLocal]?,
+        accountVotes: ReferendumAccountVotingDistribution?,
         locale: Locale
     ) -> ReferendumsSection? {
         guard let referendums else {
             return nil
         }
 
-        let tinderGovReferenda = referendums.filter { !$0.state.completed && $0.canVote }
+        let tinderGovReferenda = referendums.filter {
+            $0.canVote && (accountVotes?.votes[$0.index] == nil)
```

Comment:
One also can't vote for a referendum if the referendum is in the track in which one is delegating. So, we also need to check ```accountVotes.delegatings```